Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -175,8 +175,7 @@
             "groupName": "AdvancedDeploymentOptions",
             "options": {
                 "None": "None",
-                "ConfigureVMwithWinRM": "Configure with WinRM agent",
-                "ConfigureVMWithDGAgent": "Configure with Deployment Group agent"
+                "ConfigureVMwithWinRM": "Configure with WinRM agent"
```

Comment:
Only increment the version. Don't change the options. Any increase the version to be one greater than the release version